tap-snapshots/test/lib/commands/dist-tag.js.test.cjs

```diff
@@ -10,7 +10,7 @@ exports[`test/lib/commands/dist-tag.js TAP add new tag > should return success m
 `
 
 exports[`test/lib/commands/dist-tag.js TAP add using valid semver range as name > should return success msg 1`] = `
-dist-tag add 1.0.0 to @scoped/another@7.7.7
+add 1.0.0 to @scoped/another@7.7.7
```

Comment:
if these are logging outputs shouldn't we still expect the `dist-tag` verb?

Reply:
Ok now that I've seen the code changes this seems to be a byproduct of how the new `logs.byTitle` works?  If that's the case then you can resolve this conversation.

Reply:
I think you're right and we want these snapshots to match as close as possible to what the user sees. The byTitle thing is a quality of life testing feature to get logs with a specific prefix. But I'd like these to show the prefix too.